packages/react-google-maps-api/src/LoadScript.tsx

```diff
@@ -27,11 +29,6 @@ export function DefaultLoadingElement() {
   return <div>{`Loading...`}</div>
 }
 
-export const defaultLoadScriptProps = {
```

Comment:
@FredyC why was this extracted to utils? I think it's good co-location wise when default props are in the same place as the component

Reply:
Because of circular dependency, there was a build failure for that. And besides, the `LoadScript` will be most likely removed and replaced with LoadScriptNext in 2.0, so this is a little preparation :)

Reply:
Why LoadScript should be removed? I hear this first time

Reply:
I don't know, it's just assumption considering that `LoadScriptNext` (build on top of `useLoadScript`) is working just fine without all that clean up the mess. All fixes we have been doing lately was in this area. The `LoadScript` just seems to be broken on all fronts.